packages/minifier-css/minifier.js

```diff
@@ -65,23 +64,24 @@ const CssTools = {
    * @return {String[]} Array containing the minified CSS.
    */
   minifyCss(cssText) {
-    const f = new Future;
-    postcss([
-      cssnano({ safe: true }),
-    ]).process(cssText, {
-      from: void 0,
-    }).then(result => {
-      f.return(result.css);
-    }).catch(error => {
-      f.throw(error);
-    });
-    const minifiedCss = f.wait();
+    return Promise.await(CssTools.minifyCssAsync(cssText));
+  },
 
-    // Since this function has always returned an array, we'll wrap the
-    // minified css string in an array before returning, even though we're
-    // only ever returning one minified css string in that array (maintaining
-    // backwards compatibility).
-    return [minifiedCss];
+  /**
+   * Minify the passed in CSS string.
+   *
+   * @param {string} cssText CSS string to minify.
+   * @return {Promise<String[]>} Array containing the minified CSS.
+   */
+  async minifyCssAsync(cssText) {
+    return postcss([cssnano({ safe: true })])
+      .process(cssText, {
+        from: void 0,
+      })
+      .then((result) => [result.css])
+      .catch((error) => {
+        throw new Error(error);
+      });
```

Comment:
This could also be simplified with `Promise.await`.

Reply:
But `Promise.await` is a Fibers method. Right?

Reply:
Ah, right :facepalm: